src/Test/Utilities/Portable/Compilation/OperationTreeVerifier.cs

```diff
@@ -410,6 +410,7 @@ public override void VisitVariableDeclarator(IVariableDeclaratorOperation operat
             LogCommonPropertiesAndNewLine(operation);
 
             Visit(operation.Initializer, "Initializer");
+            VisitArray(operation.IgnoredArguments, "IgnoredArguments", logElementCount: true);
```

Comment:
>VisitArray(operation.IgnoredArguments, "IgnoredArguments", logElementCount: true); [](start = 12, length = 82)

Please consider not printing anything if there are no ignored arguments

Reply:
Done.

---
In reply to: [149217073](https://github.com/dotnet/roslyn/pull/23008#discussion_r149217073) [](ancestors = 149217073)